app/pages/learn/01_tutorial/04_mastering-the-api/02_invoke/00_methodhandle.md

```diff
@@ -0,0 +1,628 @@
+---
+id: api.invoke
+title: "Introduction to Method Handles"
+slug: learn/introduction_to_method_handles
+type: tutorial
+category: api
+category_order: 1
+layout: learn/tutorial.html
+subheader_select: tutorials
+main_css_id: learn
+description: "What is method handle mechanism, how is it different from Reflection API, and what tooling does it provide."
+author: ["NataliiaDziubenko"]
+toc:
+  - What are method handles {intro}
+  - Access checking {access}
+  - Method handle lookup {lookup}
+  - Method type {methodtype}
+  - Method handle invocation {invocation}
+  - Accessing fields {fields}
+  - Working with arrays {arrays}
+  - Exception handling {exceptions}
+  - Method handle transformations {transformations}
+  - Method Handles vs Reflection API {vsreflection}
+  - Conversion between Reflection API and method handles {unreflect}
+  - Conclusion {conclusion}
+last_update: 2024-05-04
+---
+
+<a id="intro">&nbsp;</a>
+## What are method handles
+Method handles are a low level mechanism used for method lookup and invocation. It is often compared to reflection,
+because both the Reflection API and method handles provide means to invoke methods, constructors and access fields.
+
+What exactly is a method handle? It's a direct reference to a method, constructor, or field, which can be invoked.
```

Comment:
In case of this sentence...who can invoke a method handle?

```suggestion
What exactly is a method handle? It's a direct reference to an underlying method, constructor, or field, with optional transformations of arguments or return values.
```
```suggestion
What exactly is a method handle? It's a direct reference to a method, constructor, or field, which can be invoked.
```

Reply:
Thought to add it as in the article you transform arguments of methods, but will take your suggestion too.